packages/vscode-extension/src/officeChat/utils.ts

```diff
@@ -102,5 +123,5 @@ export async function getOfficeSampleDownloadUrlInfo(sampleId: string) {
   if (!sample) {
     throw new Error("Sample not found");
   }
-  return sample.downloadUrlInfo;
+  return { downloadUrlInfo: sample.downloadUrlInfo, host: sample.types[0] };
```

Comment:
Change the function name to `getOfficeSample` and return the sample object in this method. Fetch the downloadUrlInfo and host type in outer calling function.

Reply:
done, thanks